docs/tilemap.asciidoc

```diff
@@ -25,7 +25,10 @@ Before you choose a buckets aggregation, specify if you are splitting the chart
 Coordinates* on a single chart. A multiple chart split must run before any other aggregations.
 
 Tile maps use the *Geohash* aggregation as their initial aggregation. Select a field, typically coordinates, from the
-drop-down. The *Precision* slider determines the granularity of the results displayed on the map. 
+drop-down. The *Precision* slider determines the granularity of the results displayed on the map. See the documentation 
+for the {ref}/search-aggregations-bucket-geohashgrid-aggregation.html#_cell_dimensions_at_the_equator[geohash grid] 
+aggregation for details on the area specified by each precision level. As of the 4.1 release, Kibana supports a maximum 
+geohash length of 7.
```

Comment:
You should mention that higher precisions use more memory both in the browser and in elasticsearch, also we recently merged the auto precision pull, so you should mention that is checked by default